index.html

```diff
@@ -3,314 +3,423 @@
+        {"State":"Michigan","StateAbbr":"MI","felon":0.61,"vote":19,"Gerrymandering":"26.03","PEI":1.94,"Restrictions":"Restrictions in Prison"},
+        {"State":"Ohio","StateAbbr":"OH","felon":0.61,"vote":10,"Gerrymandering":"17.22","PEI":1.54,"Restrictions":"Restrictions in Prison"},
+        {"State":"Indiana","StateAbbr":"IN","felon":0.6,"vote":13,"Gerrymandering":"41.03","PEI":-1.27,"Restrictions":"Restrictions in Prison"},
+        {"State":"Hawaii","StateAbbr":"HI","felon":0.56,"vote":7,"Gerrymandering":"8.56","PEI":"NA","Restrictions":"Restrictions in Prison"},
+        {"State":"Pennsylvania","StateAbbr":"PA","felon":0.55,"vote":8,"Gerrymandering":"17.14","PEI":1.23,"Restrictions":"Restrictions in Prison"},
+        {"State":"Illinois","StateAbbr":"IL","felon":0.52,"vote":12,"Gerrymandering":"16.64","PEI":0.4,"Restrictions":"Restrictions in Prison"},
+        {"State":"Montana","StateAbbr":"MT","felon":0.52,"vote":12,"Gerrymandering":"NA","PEI":4.51,"Restrictions":"Restrictions in Prison"},
+        {"State":"Oregon","StateAbbr":"OR","felon":0.5,"vote":"NA","Gerrymandering":"31.15","PEI":3.19,"Restrictions":"Restrictions in Prison"},
+        {"State":"Rhode Island","StateAbbr":"RI","felon":0.41,"vote":11,"Gerrymandering":"20.14","PEI":1.06,"Restrictions":"Restrictions in Prison"},
+        {"State":"Utah","StateAbbr":"UT","felon":0.4,"vote":10,"Gerrymandering":"27.66","PEI":0.34,"Restrictions":"Restrictions in Prison"},
+        {"State":"North Dakota","StateAbbr":"ND","felon":0.3,"vote":10,"Gerrymandering":"NA","PEI":-2.02,"Restrictions":"Restrictions in Prison"},
+        {"State":"New Hampshire","StateAbbr":"NH","felon":0.29,"vote":11,"Gerrymandering":"16.45","PEI":-2.36,"Restrictions":"Restrictions in Prison"},
+        {"State":"Massachusetts","StateAbbr":"MA","felon":0.25,"vote":7,"Gerrymandering":"18.45","PEI":0.19,"Restrictions":"Restrictions in Prison"},
+        {"State":"Maine","StateAbbr":"ME","felon":0,"vote":4,"Gerrymandering":"14.04","PEI":0.57,"Restrictions":"No Restrictions"},
+        {"State":"Vermont","StateAbbr":"VT","felon":0,"vote":2,"Gerrymandering":"NA","PEI":2.19,"Restrictions":"No Restrictions"}];
+
+      var MAX_DIS = 10.42;
+
+      var options = {
+        sorted: false,
+        filtered: false,
+        page: 0
+        };
+
+      function updateTable() {
+        var data = states.slice();  
+        console.log(data); 
+
+             
+
+        if(options.sorted) {
+          if (options.sorted ==='asc') {
+            data.sort(function (a,b) {return a.felon -b.felon; });
+          } else if (options.sorted==='desc') {
+            data.sort(function (a,b) {return b.felon - a.felon; });
+          }
+        }
+
+        if(options.filtered) {
+          if (options.filtered ==='no rest') {
+            data=data.filter(function (d) {return d.Restrictions==='No Restrictions'; });
```

Comment:
Thumbs up, this works as expected. It would probably be a lot simpler if `options.filtered` was `'No Restrictions'` instead of `'no rest'`, though, right?